rackcity/views/change_plan_views.py

```diff
@@ -416,7 +416,7 @@ def change_plan_execute(request, id):
             log_action(
                 request.user, updated_asset, Action.CREATE, change_plan=change_plan,
             )
-        else:
+        elif not asset_cp.is_decommissioned:
```

Comment:
only count modifications on non-decommissioned assets (so if it's modified then decommissioned in a change plan, we'll consider that a decommission, even though the modifications are still applied)